manifests/Tyrrrz/LightBulb/2.1.0.yaml

```
@@ -0,0 +1,19 @@
+Id: tyrrrz.lightbulb
```

```suggestion
 Id: Tyrrrz.LightBulb
```

Package names are generally in [PascalCase](https://techterms.com/definition/pascalcase).